generateQuestions.js

```diff
@@ -0,0 +1,15 @@
+const handleObject = require('./handleObject');
+const handleArray = require('./handleArray')
+
+// The function loops through the schema object and calls the appropriate functions based on the type of each key to eventually return a list of questions
+function generateQuestions(schema) {
+    var questionsArray = []
```

Comment:
If this isn't used outside of generateQuestions, would "let" work better?

Reply:
Great question, I pondered the same thing myself and defaulted to var given the time constraint. If I had to guess I'd say you were right given that i'm using the array with the scope it's declared in.